docs/_packages/socket_mode.md

```diff
@@ -7,133 +7,11 @@ anchor_links_header: Usage
 
 # Slack Socket Mode
 
+For baseline package documentation, please see the project's [`README`](https://github.com/slackapi/node-slack-sdk/tree/main/packages/socket-mode#readme) or the [documentation on npm](https://www.npmjs.com/package/@slack/socket-mode).
```

Comment:
Mainly axing content in here, as it used to be a straight copy/paste from the socket mode README. No need to duplicate, just link to the duplicated content and keep this area for additional documentation (if necessary).